pep/app/pods/components/search/preview/template.hbs

```diff
@@ -92,33 +92,7 @@
                                     @action={{this.viewDocument}}
                                     class="mr-2"
                                 />
-                                <DropdownMenu @horizontalPosition="auto-left" as |DD|>
-                                    <DD.Trigger @htmlTag="button" class="btn btn-link px-2">
-                                        <FaIcon @icon="download" />
-                                        {{t "search.preview.download.label"}}
-                                        <FaIcon @icon="angle-down" class="ml-1 text-muted" />
```

Comment:
i think we should make an arg in `Document::Actions::Download` to switch to this "expanded" button  content to use here, as it has the room for the more descriptive text/caret and I think it looked quite good in the original preview layout.